src/index.html

```diff
@@ -234,33 +242,6 @@ <h4 class="title is-4">
                             </div>
                         </div> <br><br>
```

Comment:
Don't use `<br>` to add margins or space between elements. Use `margin` or `padding` for that. `<br>` is for line breaks inside paragraphs `<p>` https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#Accessibility_concerns